README.md

```diff
@@ -38,26 +38,26 @@ android:name="android.support.multidex.MultiDexApplication"
 ```
 
 
-####All options
+#### All options
 
 ```groovy
 // This has to be after android{} block
 
 androidScala {
-    scalaVersion '2.11.8' 
+    scalaVersion '2.11.8'
     zincVersion '0.3.11'  // if skipped will be '0.3.11'
-    
+
     multiDex {            // helps to avoid 65k methods limit
         enabled true      // false by default
-        
+
         overwriteMainDex {
```

Comment:
long names are not very nice if you write your scripts manually
compare to this example syntax:
`
mainDex
{
  addMultidex
  addManifest
  add 'com/my/Class.class'
}`